File: periodo-1/ip/atividades/atividade-2-beecrowd/tempo-de-jogo.c
#include <stdio.h>

int main() {
    int hora_inicio, hora_fim;

    scanf("%d %d", &hora_inicio, &hora_fim);

    if (hora_inicio < hora_fim) {
        printf("O JOGO DUROU %d HORA(S)\n", hora_fim - hora_inicio);
    } else {
        printf("O JOGO DUROU %d HORA(S)\n", (hora_fim + 24) - hora_inicio);
    }

    return 0;
}

File: periodo-1/ip/atividades/atividade-4-beecrowd/tempo_de_jogo.py
hi,hf = [int(x) for x in input().split(" ")]

if (hi == hf):
    print("O JOGO DUROU 24 HORA(S)")
elif (hi > hf):
    print(f"O JOGO DUROU {(24 - hi) + hf} HORA(S)")
else:
    print(f"O JOGO DUROU {hf - hi} HORA(S)")
